File: Publisher Integration/SPX Mediation/Unified Bidding | iOS Integration/Generic/Example Ad Requests.m
SMABannerView *bannerView = [[SMABannerView alloc]init];
[SmaatoSDK prebidBannerForAdSpaceId:@"SMAATO_ADSPACE_ID" bannerSize:kSMAUbBannerSizeMediumRectangle_300x250 completion:^(SMAUbBid * _Nullable bid, NSError * _Nullable error) {
         
        if (bid && bid.metaData[@"smaato_ubid"]) {
            NSString *ubId = bid.metaData[@"smaato_ubid"];
            SMAAdRequestParams *requestParam = [[SMAAdRequestParams alloc] init];
            requestParam.ubUniqueId = ubId;
             
            bannerView.delegate = self;
            [bannerView loadWithAdSpaceId:@"SMAATO_ADSPACE_ID" adSize:kSMABannerAdSizeMediumRectangle_300x250 requestParams:requestParam];
             
        } else {
            return;
        }
    }];

File: Publisher Integration/SPX Mediation/Unified Bidding | iOS Integration/Generic/Example Ad Requests.swift
var bannerView = SMABannerView()
         
        SmaatoSDK.prebidBanner(forAdSpaceId: "SMAATO_ADSPACE_ID",
                               bannerSize: .mediumRectangle_300x250) { [unowned self] (ubBid, error) in
            guard let bid = ubBid ,
                  let bidId = bid.metaData["smaato_ubid"] as? String else {
                return
            }
             
            let requestParams = SMAAdRequestParams()
            requestParams.ubUniqueId = bidId
             
            bannerView.delegate = self
            bannerView.load(withAdSpaceId: "SMAATO_ADSPACE_ID",
                            adSize: .mediumRectangle_300x250,
                            requestParams: requestParams)
        }